src/aiortc/contrib/media.py

```diff
@@ -161,30 +207,34 @@ def __init__(self, player, kind):
         self._queue = asyncio.Queue()
         self._start = None
 
-    async def recv(self):
+    async def recv(self) -> Union[Frame, Packet]:
         if self.readyState != "live":
             raise MediaStreamError
 
         self._player._start(self)
-        frame = await self._queue.get()
-        if frame is None:
+        data = await self._queue.get()
+        if data is None:
             self.stop()
             raise MediaStreamError
-        frame_time = frame.time
+        if isinstance(data, Frame):
+            data_time = data.time
+        elif isinstance(data, Packet):
+            data_time = data.dts
```

Comment:
This doesn't seem right, the time is used for a comparison against `time.time()` so it must be in seconds. The `dts` is in an arbitrary unit, never seconds.

Reply:
Hi @jlaine.. you're right. In this case we need to `pts` (presentation timestamp) and convert it to time in seconds:
```
data_time = data.pts * (data.time_base.numerator / data.time_base.denominator)
```

I'll made this modification and test.

Reply:
I made changes. Please, check it is ok.